app/Models/Post.php

```diff
@@ -95,11 +95,11 @@ public function reactions()
     }
 
     /**
-     * Each post has one action.
+     * Each post belongs to an action.
      */
-    public function action()
+    public function actionModel()
```

Comment:
Why does this have "Model" in the name? I noticed that the other relations do not have that

Reply:
@katiecrane great q! We aren't going to remove the `action` id column yet from the `posts` table (will do this at a later date) so we have to name this relation something different to differentiate when we want to get the `action` column from the `$posts` table or the relation to get the post's `action`. e.g. to get the `action` column from the `posts` table, we'd do `$post->action` versus if we wanted to get the action model that the post belongs to, we'd do `$post->actionModel`.

Reply:
OOOO that is very tricky!! thanks for explaining!